README.md

```diff
@@ -2,40 +2,61 @@
+
+[![GitHub Actions — CI](https://github.com/PlasmaPy/PlasmaPy/workflows/CI/badge.svg)](https://github.com/PlasmaPy/PlasmaPy/actions?query=workflow%3ACI+branch%3Amaster)
+[![GitHub Actions — Style linters](https://github.com/PlasmaPy/PlasmaPy/workflows/Style%20linters/badge.svg)](https://github.com/PlasmaPy/PlasmaPy/actions?query=workflow%3AStyle-linters+branch%3Amaster)
 [![codecov](https://codecov.io/gh/PlasmaPy/PlasmaPy/branch/master/graph/badge.svg)](https://codecov.io/gh/PlasmaPy/PlasmaPy)
-[![Documentation Status](https://readthedocs.org/projects/plasmapy/badge/?version=latest)](http://plasmapy.readthedocs.io/en/latest/?badge=latest)
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/PlasmaPy/PlasmaPy/master?filepath=plasmapy%2Fexamples)
-[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
+[![Read the Docs Status](https://readthedocs.org/projects/plasmapy/badge/?version=latest&logo=twitter)](http://plasmapy.readthedocs.io/en/latest/?badge=latest)
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/PlasmaPy/PlasmaPy/master?filepath=docs/notebooks)
 
-[![Matrix](https://matrix.to/img/matrix-badge.svg)](https://app.element.io/#/room/#plasmapy:openastronomy.org)
-[![Gitter (bridged to Matrix)](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/PlasmaPy/Lobby)
-[![astropy](http://img.shields.io/badge/powered%20by-AstroPy-orange.svg?style=flat)](http://www.astropy.org/)
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1436011.svg)](https://doi.org/10.5281/zenodo.1436011)
+[![astropy](http://img.shields.io/badge/powered%20by-AstroPy-orange.svg?style=flat&logo=astropy)](http://www.astropy.org/)
+[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
 [![Open Source Helpers](https://www.codetriage.com/plasmapy/plasmapy/badges/users.svg)](https://www.codetriage.com/plasmapy/plasmapy)
 
-PlasmaPy is an open source community developed Python 3.6+ package for
-plasma physics in the early stages of development.  PlasmaPy intends to
-be for plasmas what [Astropy](https://github.com/astropy/astropy) is for
-astronomy — a collection of functionality commonly used and shared
-between plasma physicists and researchers globally, running within and
-leveraging the open source scientific Python ecosystem.  The goals of
-this project are more thoroughly described in our [vision
-statement](http://docs.plasmapy.org/en/stable/about/vision_statement.html)
-and [this recent reference](https://doi.org/10.5281/zenodo.1238132).
-We are in the process of writing [online
+[PlasmaPy](https://www.plasmapy.org/) is an open source community
+developed Python package for plasma science that is currently under
+development.PlasmaPy intends to be for plasma science what
```

Comment:
```suggestion
[PlasmaPy](https://www.plasmapy.org/) is an open source, community-developed
Python package for plasma science that is currently under
development. PlasmaPy intends to be for plasma science what
```
Incorporated suggestion from @Tiger-Du. Also, the second part of that sentence always sounded weird to me - do you expect that we'll eventually stop development?